bin/i18n/sync-in.rb

```diff
@@ -72,6 +72,15 @@ def get_i18n_strings(level)
       end
     end
 
+    # validation_strings
+    if level.validation_strings
```

Comment:
This isn't tested but should be about how to add these strings to the sync-in. Through some nice sync magic, this should actually automatically add a file called `validation_strings.<locale>.json` during the sync out.

Reply:
I added an example below of what I _think_ the sync out would produce but this would need to be tested